js/ui/components/clubhouse.js

```diff
@@ -260,9 +331,9 @@ var ClubhouseNotificationBanner = new Lang.Class({
     },
 
     _setupNextPageButton: function() {
-        let button = new St.Button({ style_class: 'notification-button',
+        let button = new St.Button({ style_class: 'notification-button-next',
                                      label: '»',
-                                     x_expand: true,
+                                     x_expand: false,
```

Comment:
If you override the `addButton` method as suggested above, then you can remove this line completely.

Reply:
+1